linux/install.sh

```diff
@@ -7,8 +7,14 @@ fc-list | grep "Bitstream Vera" > /dev/null
 RETURN=$?
 if [ $RETURN -ne 0 ];then
   echo "Bitstream Vera font family not found. Please install it:"
-  echo "sudo apt-get install ttf-bitstream-vera"
-  exit 1
+	# Detects if using Ubuntu or openSUSE, then install.
+	if [[ "$(. /etc/os-release; echo $NAME)" =~ "Ubuntu" ]]; then
+		sudo apt-get install ttf-bitstream-vera
+	fi
+	if [[ "$(. /etc/os-release; echo $NAME)" =~ "openSUSE" ]]; then
+		sudo zypper install bitstream-vera-fonts
```

Comment:
I don't want to require `sudo` in the middle of the script. Current `sudo` isn't required. The suggested command should be echoed, but not directly run.